version1/04 - Web services.md

```diff
@@ -1150,10 +1152,18 @@ Content-Type: application/json
 ###### Details
 
 - **type**
-  - object type \[Saksmappe, Mappe, Moetemappe, AbstraktMappe, Journalpost, Basisregistrering, Moeteregistrering, Arkivnotat,
-    AbstraktRegistrering, Dokument\]
+  - object type \[Arkiv, Arkivskaper, Arkivdel, Klassifikasjonssystem, Klasse, Saksmappe, Mappe, Moetemappe, Moetedeltaker, AbstraktMappe, Journalpost, Basisregistrering, Moeteregistrering, Arkivnotat,
+    AbstraktRegistrering, Dokument, Presedens, Korrespondansepart, Sakspart\]
+  - optional, but recommended
 - **id**
-  - the ID of the object
+  - the ID or system ID of the object
+  - required
+- **includeChildChanges**
+  - if true, marker events for creates/updates/deletes will be returned for supported entities
```

Comment:
```suggestion
  - if true, marker events for creates/moves/deletes will be returned for supported entities
```

Reply:
Technically, child _updates_ could be included as well. For example, correspondence party name change when fetching a registry entry's changelog.

Reply:
But aren't these kind of changes reserved for the `nestChildChanges` flag?

Reply:
They are 😅 . Applying your suggestion